Authentication/Login/View/LoginView.swift

```diff
@@ -37,11 +38,26 @@ public extension LoginViewType {
 /** Default login view. */
 internal final class LoginView: UIView, LoginViewType, NibLoadable {
     
+    // - Warning: This delegate must be set before calling the `render` function.
```

Comment:
The thing is that there is no error. It just uses the default instead of your wanted delegate. And we can't know if it was set by the user or not, or if the user preferred to use the default and so didn't set it. When and which message would you log?
(I am adding an explanation of what would happen if not in this comment nonetheless.)